custom/bihar/management/commands/bihar_cleanup_case_2.py

```diff
@@ -0,0 +1,70 @@
+import csv
+from django.core.management import BaseCommand
+from casexml.apps.case.models import CommCareCase
+from corehq.apps.hqcase.utils import get_case_by_identifier
+
+
+MOTECH_ID = "fb6e0b19cbe3ef683a10c4c4766a1ef3"
+
+
+class Command(BaseCommand):
+    """
+    One time command for cleaning up care-bihar data.
+    Takes one argument, the directory containing the three input files.
+    """
+
+    cases_to_save = {}
+
+    def get_case(self, case_id):
+        case = self.cases_to_save.get(case_id) or get_case_by_identifier("care-bihar", case_id)
```

Comment:
Checked the ticket and I'm 100% sure you should just use `CommCareCase.get` which will prevent hitting ES twice as well as couch for every case.